apps/cb-ceiling/server/dataFunctions/data_series.js

```diff
@@ -45,34 +45,19 @@ dataSeries = function (plotParams, plotFunction) {
   let ymin = Number.MAX_VALUE;
   const idealValues = [];
   let statement = "";
+  var queryTemplate = null;
 
   for (let curveIndex = 0; curveIndex < curvesLength; curveIndex++) {
     // initialize variables specific to each curve
 
     const curve = curves[curveIndex];
     const regionType = curve["region-type"];
-    let queryTemplate = null;
-    if (regionType === "Predefined region") {
-      queryTemplate = fs.readFileSync(
-        "assets/app/sqlTemplates/tmpl_TimeSeries_region.sql",
-        "utf8"
-      );
-    } else {
-      queryTemplate = fs.readFileSync(
-        "assets/app/sqlTemplates/tmpl_TimeSeries_stations.sql",
-        "utf8"
-      );
-    }
-
-    queryTemplate = queryTemplate.replace(/vxFROM_SECS/g, fromSecs);
-    queryTemplate = queryTemplate.replace(/vxTO_SECS/g, toSecs);
-
+    var queryTemplate = null;
```

Comment:
Why is queryTemplate re-declared instead of just being reassigned?

Reply:
@randytpierce good catch, fixed!